GetandTake/Services/Concretes/ProductManager.cs

```diff
@@ -0,0 +1,77 @@
+﻿using AutoMapper;
+using GetandTake.DataAccess.Repositories;
+using GetandTake.Models;
+using GetandTake.Models.DTOs.BaseDTO;
+using GetandTake.Models.DTOs.ListDTO;
+using GetandTake.Services.Abstracts;
+using Microsoft.EntityFrameworkCore;
+
+namespace GetandTake.Services.Concretes
+{
+    public class ProductManager : IProductService
+    {
+        private readonly IEntityRepository<Product> _repository;
+        private readonly IMapper _mapper;
+        public ProductManager(IEntityRepository<Product> repository, IMapper mapper)
+        {
+            _repository = repository;
+            _mapper = mapper;
+        }
+
+        public async Task<IEnumerable<ProductsDTO>> GetAllAsync()
+        {
+            var products = _repository.AsNoTracking
+                .Include(p => p.Category)
+                .Include(p => p.Supplier);
+            return _mapper.Map<List<ProductsDTO>>(products);
```

Comment:
Try to **always** add empty space before `return` statement, it is a good practice and it will simplify finding and reading returnable value of method